app/build.gradle.kts

```diff
@@ -504,6 +505,9 @@ apply(plugin = "com.google.gms.google-services")
 
 abstract class TransformGeneratedRoomDaoTask : DefaultTask() {
 
+  @get:Input
+  abstract val projectDirAbsolutePath: Property<String>
```

Comment:
Since this property doesn't change, instead of passing it as parameter. We can just have the variable inside this task. WDYT?

Reply:
```kt
private val projectDir = project.projectDir.absolutePath
```

Reply:
Fixed